### ISSUE TITLE
Request for pretrained model/weight

### ISSUE THREAD

**Issue by breadxiexie** [hridayK/Detection-of-AI-generated-images#5]
Thanks for your work.But I got some problems about test.My model didn't work well on some pictures.Can you share your pretrained model/weight? Thank you.

**Comment by hridayK** [comment]
I don't have the resources to train the model, but while attempting to train the model I had missed out a few details which you may want to take note of, the authors of the paper seemed to have trained their model on a dataset of PNG images. I was unable to find any dataset of AI generated images vs Real images that contained PNG images, so even if there were any occasional PNG images in any dataset I convert them to JPEG. (I had mentioned a high validation accuracy in the README.md but another mistake during that part was keeping a really small validation set which inaccurately showed high validation accuracy in early stages of training).

![image](https://github.com/hridayK/Detection-of-AI-generated-images/assets/16136188/db9275ab-bad1-4220-980c-d4e4f54ac160)

Here is the performance measured by the author on JPEG images after training, in case you have the resources to train the model and if you can find a PNG dataset regarding the same, I'll be happy to work with you in customizing the code for PNG images and training it under the same conditions as the authors so that the model weights can be made public.

**Comment by breadxiexie** [comment]
The dataset employed in the original paper presumably consists exclusively of PNG formatted images. Furthermore, I am aware that the AI-generated images within the genimage dataset all bear the PNG file extension. The repository I located on GitHub hosting the dataset utilized in the original paper can be found  at:https://github.com/PeterWang512/CNNDetection?tab=readme-ov-file         Additionally, could you provide the model that has been trained on the test set initially? Despite any potential errors, it still holds considerable reference value. Could you also share the test code? I wish to ensure that there are no major issues with my testing approach.

**Comment by hridayK** [comment]
sorry for the late reply, I took hours to train for a single epoch and in the end it ended up being a random guessing model with poor accuracy, which I had deleted eventually.... the pretrained one will not be of much help. I can help modify the current code for training on png images since the current preprocessing method converts pngs to jpegs because there were a few pngs in the datasets used.

**Comment by breadxiexie** [comment]
OK，thanks for your work.I think maybe the way of paper have some problem on the dataset.

**Comment by hridayK** [comment]
You are using their benchmarking dataset, just checked the link. The mentioned dataset is actually used for training another model against which the authors compare their own model.
